web/src/components/NumberField.test.js

```diff
@@ -20,6 +20,7 @@ describe('NumberField component', () => {
     ).toMatchSnapshot();
   });
 
+  // should text be selected if 'min' is set, and value <= min ?
```

Comment:
Should it?

Reply:
I think the functionality should still work for zero, even if the min is not set.  So I would say leave it "as is" rather than change it to select min if min is set.

Reply:
**todo**: delete this comment